### DISCUSSION TITLE
Switching to older version should keep open page

### DISCUSSION THREAD

**Discussion by kiru** [tailwindlabs/tailwindcss#9672] [General]
Hello, 

This is such a huge annoyance. 

1. You search on Google: "tailwind cursor". 
2. You land on https://tailwindcss.com/docs/cursor
3. You want version 2 
4. You switch to version 2
<img width="408" alt="CleanShot 2022-10-27 at 15 56 20@2x" src="https://user-images.githubusercontent.com/2451608/198304623-90db3509-2e43-4e8e-96c7-043e943961b2.png">
5. You go to https://v2.tailwindcss.com/
6. You CTRL-K and search cursor
7. You are back on https://v2.tailwindcss.com/docs/cursor

Is it possible to skip 4-6? When I click on the older version, is it possible to "just" append "v2." to the url? 

Thank you, 
Kiru

**Comment by Amyconnell** [comment]
Open your phone's Settings app. Near the bottom, tap System. System update. You'll see your update status. Follow any steps on the screen.
https://www.myccpay.vip

**Reply by kiru** [reply]
Can you elaborate? Which phone? 
The above description mostly applies for the Desktop version of the webpage.
